lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -469,6 +469,8 @@ impl Dom {
 pub struct DomLayers {
     /// Back DOM scene layer.
     pub back:           DomScene,
+    /// Back DOM scene layer with disabled panning.
```

Comment:
Does this documentation explain what is this layer? Is this really a back-layer? Is it meant to be behind graph editor with disabled panning? So do you want to store there things on top of which you want to display nodes?

Reply:
> Does this documentation explain what is this layer? Is this really a back-layer? Is it meant to be behind graph editor with disabled panning? So do you want to store there things on top of which you want to display nodes?

I think the answer is: to see our cursor over the panels. The `fullscreen_vis` panel is behind the scene for the same reason. It should be added to the comment.

Reply:
Exactly, I described my thoughts on z-indexing here: https://github.com/enso-org/enso/pull/3159#discussion_r752067615
I'll add it to the comment

Reply:
Yep, this makes sense. This explanation should be part of the doc.

Reply:
Explanation added to the doc